docs/releases/contributors.rst

```diff
@@ -0,0 +1,29 @@
+
+Contributors
+------------
+
+This page lists all people who have contributed to the development of
+``datatable``. We take into account both code and documentation contributions,
+as well as contributions in the form of bug reports and feature requests.
+
+More specifically, a *code contribution* is considered any PR (pull request)
+that was merged into the codebase. The "complexity" of the PR is not taken
+into account as it is highly subjective. Next, an *issue contribution* is
+any closed issue except for those that are tagged as "question", "wont-fix"
+or "cannot-reproduce". Issues are attributed according to their closing date,
+not their creation date.
+
+In the table, the contributors are sorted according to their total contribution
+score, which is the weighted sum of the count of each user's code and issue
+contributions. Code contributions have more weight than issue contributions,
+and more recent contributions more weight than the older ones.
+
+.. contributors-grid::
+
+
+**Developer's note**: This table is auto-generated based on contributor lists
+in each of the version files, specified via the ``..contributors::`` directive.
+In turn, the list of contributors for each version has to be generated via
+the script ``ci/gh.py`` at the time of each release. The issues/PRs will be
```

Comment:
btw, why such a mysterious name `gh.py`? :)

Reply:
Short for "github".  The script, however, requires `github` python package, and I didn't want to name the script same way as an existing module since it might cause problems during importing.